include/inexor/vulkan-renderer/availability_checks.hpp

```diff
@@ -76,15 +64,14 @@ class AvailabilityChecksManager {
 
     /// @brief Checks if presentation is available for a certain combination of graphics card and window surface.
     /// The present mode describes how the rendered image will be presented on the screen.
-    /// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkGetPhysicalDeviceSurfaceSupportKHR.html
-    /// @param graphics_card [in] The selected graphics card.
-    /// @param surface [surface] The window surface.
-    /// @return true if presentation is available, false otherwise.
+    /// @param graphics_card The selected graphics card.
+    /// @param surface The window surface.
+    /// @return 'true' if presentation is available, 'false' otherwise.
     [[nodiscard]] bool has_presentation(const VkPhysicalDevice &graphics_card, const VkSurfaceKHR &surface);
 
     /// @brief Checks if swapchain is available for a certain graphics card.
-    /// @param graphics_card [in] The selected graphics card.
-    /// @return true if swapchain is available, false otherwise.
+    /// @param graphics_card The selected graphics card.
+    /// @return 'true' if swapchain is available, 'false' otherwise.
```

Comment:
```suggestion
    /// @return ``true`` if swapchain is available, ``false`` otherwise.
```

Reply:
Done. I removed the `false otherwise` everywhere.